### ISSUE TITLE
[Accordion]: WCAG problems

### ISSUE THREAD

**Issue by mart-sessman** [TEDI-Design-System/angular#261]
## Problem
* Hidden content is focusable
* Screen-reader reads hidden content
* focus state for accordion header missing

## Solution
should disable focusing on hidden accordion content and hide it from screenreader using correct aria tags

## Project
SKAIS
